modules/migrations/base/options.go

```diff
@@ -8,4 +8,27 @@ package base
 import "code.gitea.io/gitea/modules/structs"
 
 // MigrateOptions defines the way a repository gets migrated
-type MigrateOptions = structs.MigrateRepoOption
+type MigrateOptions struct {
+	// required: true
+	CloneAddr    string `json:"clone_addr" binding:"Required"`
+	AuthUsername string `json:"auth_username"`
+	AuthPassword string `json:"auth_password"`
+	AuthToken    string `json:"auth_token"`
+	// required: true
+	UID int `json:"uid" binding:"Required"`
+	// required: true
+	RepoName        string `json:"repo_name" binding:"Required"`
+	Mirror          bool   `json:"mirror"`
+	Private         bool   `json:"private"`
+	Description     string `json:"description"`
+	OriginalURL     string
+	GitServiceType  structs.GitServiceType
+	Wiki            bool
+	Issues          bool
+	Milestones      bool
+	Labels          bool
+	Releases        bool
+	Comments        bool
+	PullRequests    bool
+	MigrateToRepoID int64
```

Comment:
without `json:...` these will be mapped as they are e.g. "MigrateToRepoID":

Reply:
but I guess these now form part of an API?

Reply:
@zeripath just look at f66c2e025ebcb7906904636e454140a119f51867 's message

-> this struct is for internal usage by migration module - and those func who interact with it

I'll add some comments ...